src/window/city.c

```diff
@@ -155,6 +161,100 @@ static void cycle_legion(void)
     }
 }
 
+/** 
+    Takes a building and retrieve its proper type for cloning.
+    For example, given a fort, return the enumaration value corresponding to
+    the specific type of fort rather than the general value
+
+    @param building Building to examine
+    @return the building_type value to clone, or BUILDING_NONE if not cloneable
+*/
```

Comment:
For documentation comments, please use the indentation you also see in header files: start every line with ` * ` rather than indenting by 4 spaces.